docs/src/_cloud/onboarding.md

```diff
@@ -12,13 +12,19 @@ weight: 2
 
 ## Prereqs
 
-### Prereq #1: Provide access to your repo
+### Prereq #1: Sign up for the waitlist
+
+While Meltano Cloud is in Beta, you must sign up via [the waitlist](https://meltano.com/cloud/) in order to gain access.
+
+Waitlist members are added to Meltano Cloud on a rolling basis. You'll receive an email once you've been added.
+
+### Prereq #2: Provide access to your repo
 
 Click the following link to install the [Meltano Cloud GitHub App](https://github.com/apps/meltano-cloud) to your organization.
 When asked "Where do you want to install this app?" you _must_ select your GitHub organization, not your personal account.
 When asked which repositories to provide the app access to, select your GitHub project repo.
 
-### Prereq #2: Creating a test or sandbox environment
```

Comment:
Is this step required by users? It seems like more of a best practice than a prereq if its a recommendation. Although the line "..can be used by Meltano Cloud to test that all jobs are working as expected" makes me think that Meltano Cloud is doing something automated during onboarding to tests that jobs and schedules run. I would guess we're not doing anything fancy yet though.

Reply:
I think we could move this to a different section (maybe in a separate PR). If they're onboarding to cloud they presumably already have a project so they may not need to do this.

Reply:
@pnadolny13 @tayloramurphy thanks for the callout. In our private beta onboarding, I know a lot of customers didn't actually have a project ready when they began onboarding. I was thinking this would also likely be the case moving forward, so chose to leave this in. What do you think about maybe removing this section and instead just linking to our "Getting started" docs for Meltano core? I think it's worth calling out that you need to actually have a Meltano project set up in order to run it on Cloud but I also think that our existing Meltano core docs provide a better walkthrough of that.